tests/unit/stage/test_cache.py

```diff
@@ -204,3 +206,24 @@ def _mode(path):
     assert _mode(parent_cache_dir) == dir_mode
     assert _mode(cache_dir) == dir_mode
     assert _mode(cache_file) == file_mode
+
+
+def test_always_changed(mocker):
+    from dvc.repo import Repo
+    from dvc.stage.cache import RunCacheNotFoundError, StageCache
+
+    repo = mocker.Mock(spec=Repo)
+    cache = StageCache(repo)
+
+    class StageMock:
```

Comment:
I think it'd be better to create a real stage.

Reply:
Good point! Esp if we expect type checks in the near future ;) Thanks!